README.md

```diff
@@ -2,31 +2,122 @@
 
 ## Learning Goals
 
-- Learning Goal 1
-- Learning Goal 2
+- Introduce different types of object relationships
+- Introduce Unified Modeling Language (UML)
 
 ## Introduction
 
-A 1-2 sentence summary of what will be covered.
+Objects are often related to other objects.  There are
+several types of relationships.  
 
-## Topic 1
+- *Association* represents a general relationship between two objects,
+  where neither object is considered a part or member of the
+  other.
+- *Aggregation* represents a whole-part or parent-child relationship
+  between objects.  Aggregation represents a *weak* existence relationship,
+  where the child object can exist independently of the parent.
+- *Composition* also represents a whole-part or parent-child relationship.
+  However, the child object can not exist independently of the whole/part.
+  Composition represents a *strong* existence relationship. Deletion of
+  the parent would result in the child deletion as well.
 
-Discuss the topic of the reading. Break into sections and sub-sections as
-appropriate to help with the reader's orientation with the material and future
-referencing.
+## Unified Modeling Language (UML)
 
-## Topic 2
+**Unified Modeling Language (UML)** is a standardized modeling language to visually
+represent software artifacts.    We can create UML diagrams to depict class structure
+and class relationships.  We draw a class as shown below.  The fields and methods may
+be omitted.  Relationships between classes are drawn using different types of edges.
+
+![uml class](https://curriculum-content.s3.amazonaws.com/6676/java-multipleclasses/uml_class.png)
+
+## Association
+
+
+**Association** is used to model a general relationship between objects.
```

Comment:
Looks like the other terms below are italicized instead of bolded. I'd do the same here to be consistent.

Reply:
fixed